core/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id("org.jetbrains.kotlin.jvm") version "1.9.22"
+}
+
+group = "org.holoeasy"
+version = "3.1.1"
```

Comment:
```suggestion
```
The submodule does not need to repeat this content as it will automatically retrieve the parent project's